### PR TITLE
Move file operations to S3

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: test/unit/services/workers/test-generate-payout-payments.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ describe('services/workers/generate-payout-payments', function () {
       expect(getClaimsPendingPayment.calledOnce).to.be.true //eslint-disable-line
       expect(getTopUpsPendingPayment.calledOnce).to.be.true //eslint-disable-line
       expect(createPayoutFile.calledWith(claimsPendingPayment)).to.be.true //eslint-disable-line
-      expect(sftpSendPayoutPaymentFile.calledWith(testPath, './test.csv')).to.be.true //eslint-disable-line
+      expect(sftpSendPayoutPaymentFile.calledWith(`/tmp/${testPath}`, './test.csv')).to.be.true //eslint-disable-line
```
Comment (andrewrlee) [inline test/unit/services/workers/test-generate-payout-payments.js:73]: does this need updating? guess it's not sftp anymore?

File: app/services/cleanup-old-data/delete-old-files.js [hunk 2]
```diff
@@ -1,22 +1,14 @@
...
-          try {
-            fs.unlinkSync(document.Filepath)
-          } catch (error) {
-            if (error.code === 'ENOENT') {
-              log.error(`File for Claim ${claimId} with path ${document.Filepath} does not exist`)
-            } else {
-              throw error
-            }
-          }
+          aws.delete(document.Filepath)
```
Comment (andrewrlee) [inline app/services/cleanup-old-data/delete-old-files.js:11]: think this returns a promise, won't this swallow any errors?

Comment (andrewrlee) [reply app/services/cleanup-old-data/delete-old-files.js:11]: or rather maybe it should return a promise to allow the same error behaviour as before?

File: test/unit/services/workers/test-generate-payout-payments.js (hunk 1 above)
Comment (psoleckimoj) [reply test/unit/services/workers/test-generate-payout-payments.js:73]: No, the files are still SFTPed

File: app/services/cleanup-old-data/delete-old-files.js (hunk 2 above)
Comment (psoleckimoj) [reply app/services/cleanup-old-data/delete-old-files.js:11]: I'm currently rethinking my understanding of the Promise stuff anyway so I might just delete my post and ask again later :)

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**